db/mysql/client.go

```diff
@@ -5,6 +5,7 @@ import (
 	"database/sql"
 	"fmt"
 
+	// The following import is required to register the MySQL driver with the database/sql package.
 	_ "github.com/go-sql-driver/mysql"
 	wraperrors "github.com/pkg/errors"
```

Comment:
remove this wrapper package, golang standard library supports this feature 
use fmt package  instead of 
wraperrors.Wrap(err, "message") use 
 fmt.Errorf("message: %w", err) 
 
this changes should be reflrected in db/mysql/client.go file and all of the service layer

Reply:
Removed and changed in the service layer as well